uefi_app/SbsaAvsMain.c

```diff
@@ -455,8 +455,10 @@ ShellAppMainsbsa (
   */
   configureGicIts();
 
-  Print(L"\n      *** Starting PCIe Exerciser tests ***  \n");
-  Status |= val_exerciser_execute_tests(g_sbsa_level);
+  if (g_sbsa_level > 3) {
```

Comment:
Exerciser tests are applicable from level 3 and above. So, this change is not required.

Reply:
val/src/avs_exerciser.c has this:

```
  if (level <= 3) {
    val_print(AVS_PRINT_WARN, "Exerciser Sbsa compliance is only from Level %d \n", 4);
    return AVS_STATUS_SKIP;
  }  ```

I just moved check into other place.